etc/dockerfiles/dookug-release.Dockerfile

```diff
@@ -0,0 +1,102 @@
+ARG WILDFLY_BASE_IMAGE
+ARG ARTIFACT_DOWNLOADER_IMAGE
+
+ARG DOWNLOAD_DIR_NAME="download"
+
+################################################################################
+# Default image customization
+################################################################################
+FROM ${WILDFLY_BASE_IMAGE} AS base
+
+#COPY --chown=jboss:jboss etc/jboss-cli/*.cli $HOME/config
+#COPY --chown=jboss:jboss etc/jboss-cli/embed-server.cli $HOME/config/embed-server.cli
+#COPY --chown=jboss:jboss etc/jboss-cli/cli.properties $HOME/config
+
+#RUN $WILDFLY_HOME/bin/jboss-cli.sh \
+#        --properties=$HOME/config/cli.properties \
+#        --file=$HOME/config/embed-server.cli \
+#    && rm -rf $WILDFLY_HOME/standalone/configuration/standalone_xml_history/current/*
+
+
+################################################################################
+# Download .war
+################################################################################
+FROM ${ARTIFACT_DOWNLOADER_IMAGE} AS download
+ENV LANG='en_US.UTF-8' LANGUAGE='en_US:en'
+
+ARG POM_GROUP_ID
+ARG POM_ARTIFACT_ID
+ARG POM_VERSION
+ARG POM_EXTENSION
+
+ENV NEXUS_OBJECT_GROUP_ID=$POM_GROUP_ID
+ENV NEXUS_OBJECT_ARTIFACT_ID=$POM_ARTIFACT_ID
+ENV NEXUS_OBJECT_VERSION=$POM_VERSION
+ENV NEXUS_OBJECT_EXTENSION=$POM_EXTENSION
+
+ENV NEXUS_DOWNLOAD_OUTPUT_FILE_NAME=$NEXUS_OBJECT_ARTIFACT_ID.$NEXUS_OBJECT_EXTENSION
+
+RUN echo "DOWNLOAD_DIR=$DOWNLOAD_DIR" && \
+    ${HOME}/script/maven-search-download.sh
+
+################################################################################
+# Create production image
+################################################################################
+FROM base AS prod
+
+ARG POM_ARTIFACT_ID
+ARG POM_VERSION
+ARG POM_EXTENSION
+ARG COFFEE_MODEL_BASE_JAVA_TIME_TIMEZONE_ID="UTC"
+
+ARG DOWNLOAD_DIR_NAME
+ARG DOWNLOAD_DIR=$HOME/$DOWNLOAD_DIR_NAME
+ARG PROPERTIES_FILE
+
+ENV LOGSTASH_MODULE_ID="$POM_ARTIFACT_ID"
+ENV LOGSTASH_MODULE_VERSION="$POM_VERSION"
+
+ENV HIBERNATE_DIALECT=$DEFAULT_PERSISTENCE_HIBERNATE_DIALECT
+ENV COFFEE_MODEL_BASE_JAVA_TIME_TIMEZONE_ID="$COFFEE_MODEL_BASE_JAVA_TIME_TIMEZONE_ID"
+
+ARG DEFAULT_FONT=Roboto
+ARG DEFAULT_FONTS_DIRECTORY=etc/docker-compose/fonts
```

Comment:
ha már a fonts dir törölve lett, akkor itt is feleslegesek ezek nem?

Reply:
csak a roboto-n kivuli fontok lettek torolve